pkg/vsphere/vm/vm.go

```diff
@@ -272,17 +272,26 @@ func (vm *VirtualMachine) DeleteExceptDisks(ctx context.Context) (*types.TaskInf
 
 	op := trace.FromContext(ctx, "DeleteExceptDisks")
 
+	op.Debugf("Getting list of the devices for VM %s", vm)
 	devices, err := vm.Device(op)
 	if err != nil {
 		return nil, err
 	}
 
 	disks := devices.SelectByType(&types.VirtualDisk{})
-	err = vm.RemoveDevice(op, true, disks...)
+	op.Debug("Removing devices for VM")
```

Comment:
I know that we _can_ correlate log messages using the operation ID, but is there any reason not to include the VM name on each of these log lines? I think that would improve readability in cases where we have concurrent operations.

I haven't seen a logging best practices document for our codebase, but this seems like the sort of thing we'd want to document and be consistent about. If the established pattern is to just include the name once, just ignore this suggestion.

Reply:
plus that. In vic-machine-server log if we have concurrent operations the logs are mingled. The log message that actually includes the VCH name/ID will interleave with all other log messages from different operations. Even tho searching for op ID works, being able to navigate thru the changes of a specific VCH in the log definitely helps.

Reply:
@zjs I had VM name in each log before, but it feels redundant in case we have op id. I also have big plans to keep all necessary data within context so logs will automatically contain all relevant information, devs will need to log a log message only or they also may be able to add more info to get logged.